lib/ansiblelint/cli.py

```diff
@@ -98,6 +98,9 @@ def get_cli_parser():
                            "rules directories specified with -r. There is "
                            "no need to specify this if no -r flags are used"
                            % ansiblelint.default_rulesdir)
+    parser.add_option('-s', dest='relative_path', action='store_false',
```

Comment:
Why `-s`? How does it relate to paths?

Reply:
No clue... did I cut paste something ? :thinking: 
Honestly, I don't really have an opinion on this so if you want to pick on or only use a long option name... please be my guest, choose something.

Reply:
@ssbarnea do you think this should be configurable?

Reply:
Most options changing the output have long names: `--nocolor`, `--force-color`. We could go from something like `--show-relpath` (changed, but still open to suggestion).

Reply:
This is probably fine